x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -523,3 +514,23 @@ const handleIsolateEndpointHost = async (
     });
   }
 };
+
+async function getEndpointPackageInfo(
+  state: ImmutableObject<EndpointState>,
+  dispatch: Dispatch<ServerReturnedEndpointPackageInfo>,
+  coreStart: CoreStart
+) {
+  if (endpointPackageInfo(state)) return;
```

Comment:
its probably unlikely (probably 😄 ), but the package version could change if one installed between checks. Not a big deal since URL that uses this version will still be valid, but wanted to mention it.

Reply:
that's a good point.